src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
you should use `toLowerCase()` instead of `toString()` since `toString()` will just give you a string representation of the constant - `WHITE` for example, however you need `white`
```suggestion
        return Color.values()[index].toLowerCase();
```